warehouse/snowflake/snowflake.go

```diff
@@ -132,9 +132,7 @@ func ColumnsWithDataTypes(columns map[string]string, prefix string) string {
 
 // schemaIdentifier returns [DATABASE_NAME].[NAMESPACE] format to access the schema directly.
```

Comment:
Please modify the comment as per the changes.

Reply:
Thanks for pointing. Will update this later.